config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -125,3 +125,98 @@ periodics:
         requests:
           cpu: 2
           memory: "6Gi"
+
+- name: ci-kubernetes-e2e-gci-gce-scalability-watch-list-off
+  cluster: k8s-infra-prow-build
+  interval: 6min
+  tags:
+    - "perfDashPrefix: watch-list-off"
+    - "perfDashJobType: watchlist-off"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-common: "true"
+    preset-e2e-scalability-periodics: "true"
+    preset-e2e-scalability-periodics-master: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: watchlist-off
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230406-23cb1879e3-master
+        command:
+          - runner.sh
+          - /workspace/scenarios/kubernetes_e2e.py
+        args:
+          - --check-leaked-resources
+          - --env=CONTAINER_IMAGE=registry-sandbox.k8s.io/pause:3.1 #TODO(ameukam): revert when registry.k8s.io is ready
+          - --extract=ci/latest
+          - --gcp-node-image=gci
+          - --gcp-nodes=1
+          - --provider=gce
+          - --test=false
+          - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+          - --test-cmd-args=cluster-loader2
+          - --test-cmd-args=--nodes=1
+          - --test-cmd-args=--provider=gce
+          - --test-cmd-args=--report-dir=$(ARTIFACTS)
+          - --test-cmd-args=--testconfig=testing/watch-list/config.yaml
+          - --test-cmd-name=ClusterLoaderV2
+          - --timeout=6m
```

Comment:
I defined it to match the `interval`

Reply:
let's increase it - let's don't try to overoptimize and later debug because something took a bit longer.

30m?

Reply:
I defined it to 6m because the test's timeout is 5 min. But you are right, scheduling might take longer and it would fail. I'll increase it to 30m.